com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -300,6 +313,13 @@ public static void NetworkHide(List<NetworkObject> networkObjects, ulong clientI
                 throw new VisibilityChangeException("Cannot hide an object from the server");
             }
 
+            if (networkObjects == null || networkObjects.Count == 0)
+            {
+                throw new ArgumentNullException("At least one NetworkObject has to be provided");
+            }
+
+            NetworkManager networkManager = networkObjects[0].NetworkManager;
```

Comment:
Due to the minor-hoops you have to jump through here for hiding a list of NetworkObjects, I was wondering if this merits the potential of moving the NetworkHide methods into a different class so you can still keep separation from the NetworkObjects being hidden or shown but don't have to rely upon the Singleton?  (unless you have other plans to handle this later?)